### ISSUE TITLE
CMD + r | CMD + d not works as expect - RN version "0.61.0"

### ISSUE THREAD

**Issue by mustafaskyer** [facebook/react-native#28552]
## Description
Hello,
I noticed that in react-native: 0.61.0, the cmd + r & cmd + d which should reload the app, or open the dev menu not work as expected,
it works for the first time after so it doesn't work

## what I tried before submitting this issue ?
- reset cache, clear watchman
- removed pods & podfile.lock and re-installed
- followed all solutions [here](https://github.com/facebook/react-native/issues/306) one by one

## React Native version:
```js
info Fetching system and libraries information...
System:
    OS: macOS 10.15.4
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 47.88 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 13.7.0 - /usr/local/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.13.6 - /usr/local/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-23 | Google APIs Intel x86 Atom, android-28 | Intel x86 Atom_64, android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.6010548
    Xcode: 11.4/11E146 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.0 => 0.61.0 
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. run app
2. try run cmd + r || cmd + d not works as expected

## Expected Results
- opening dev menu onPress CMD + d
- reload app onPress CMD + r

**Comment by memosize** [comment]
lol  even 0.62.2 not working again

**Comment by mustafaskyer** [comment]
@memosize for me, I updated from *"0.61.0"* to *" 0.62.1"* and it works as expected

**Comment by andidev** [comment]
I have the same problem. This is just a guess but I have recently updated xcode to the latest version. Could it be that? Using version `Version 11.4 (11E146)`

**Comment by andidev** [comment]
I think that a workaround may be running an old simulator by passing the version in parentheses e.g. (12.1) worked for me.

e.g.
```
react-native run-ios --simulator 'iPhone 8 (12.1)'
```

**Comment by sregg** [comment]
This started when I updated to Xcode 11.4 and simulators are on iOS 13.4

**Comment by CesarDenis** [comment]
The same with me, started after updated to Xcode 11.4 and simulators are on iOS 13.4 and 12.4.

**Comment by mustafaskyer** [comment]
if anyone still faces this issue, as @andidev said, just downgrade simulator ios version under 13.

**Comment by rickhanlonii** [comment]
Hey all, thanks for the report!

From what I can tell, this is an Apple bug in iOS 13+ where hardware keyboards do not send the correct `cmd + <key>` event until the key has been used without command. I've reproduced this behavior in every iOS 13 version using RNTester.

To verify, I overrode the `UIApplication.handleKeyUIEvent` method which is called when a hardware key is pressed and confirmed that only the "command" event is fired for `cmd + r`. We do not get an event for the "r" press until `r` has been pressed on its own first. If you press `r` and then `cmd+r` then the hotkey reliably works for the rest of the life of the app.

We do some hacky swizzling to attach the key commands, so just to be sure that this isn't caused by the hack, I ran a test using the actual documented [keyCommands](https://developer.apple.com/documentation/uikit/uikeycommand?language=objc) UIResponder method that's we're supposed to be using and it has the exact same issue: the `cmd+<key>` is not called until the `<key>` has been pressed on its own.

This applies to all in-app keys such as `cmd+r`, `cmd+d`, and `cmd+i`.

Finally, I looked at [FLEX](https://github.com/Flipboard/FLEX) which also registers global hotkeys but in a more robust way. As a last ditch effort, I copied their hotkey implementation over to our `RCTKeyCommands` and it still contains the bug. FLEX doesn't seem to be impacted because they do not use `cmd+<key>`, they only use `<key>` alone such as `f` and `n`.

The video below shows the bug on iOS 13 but not iOS 12. Notice that the first 4 `cmd+r` attempts fail, then I press `r` alone, and then `cmd+r` works. When I switch to the device on the left running iOS 12, it always works.


![out](https://user-images.githubusercontent.com/2440089/81414905-b24db800-9115-11ea-8fa1-bedcd170d9e0.gif)
